### DISCUSSION TITLE
I accidentally unpaired my active sensor. How to get going again?

### DISCUSSION THREAD

**Discussion by gitanO0** [NightscoutFoundation/xDrip#3404] [General]
As the title says, how do I get an active sensor/transmitter going again when I accidentally unpaired the transmitter from my phone. G6, Android. I'm on 126 minutes of no connections.

**Comment by Navid200** [comment]
This is not a problem at all.
Try this first and let me know if it doesn't work: https://navid200.github.io/xDrip/docs/MissedPairRequest.html

**Reply by gitanO0** [reply]
Ok thx. Will try that and report back.

**Reply by gitanO0** [reply]
My goodness that was easy. Thanks.   Wish more problems in life were that easy to fix. 😀

**Reply by Navid200** [reply]
Great
